### ISSUE TITLE
test pretrained checkpoint

### ISSUE THREAD

**Issue by chenwydj** [SysCV/qdtrack#73]
Dear authors,

I am trying to test the pretrained bdd100k model using

```bash
CUDA_VISIBLE_DEVICES=0 python tools/test.py configs/bdd100k/qdtrack-frcnn_r50_fpn_12e_bdd100k.py \
qdtrack-frcnn_r50_fpn_12e_bdd100k-13328aed.pth \
--eval bbox --show
```

However, it shows an error as below. Could you help or give any suggestion? Thank you very much!
```bash
loading annotations into memory...
Done (t=4.83s)
creating index...
index created!
/usr/local/lib/python3.6/dist-packages/mmdet/core/anchor/builder.py:16: UserWarning: ``build_anchor_generator`` would be deprecated soon, please use ``build_prior_generator``
  '``build_anchor_generator`` would be deprecated soon, please use '
Use load_from_local loader
completed: 0, elapsed: 0sTraceback (most recent call last):
  File "tools/test.py", line 163, in <module>
    main()
  File "tools/test.py", line 159, in main
    print(dataset.evaluate(outputs, **eval_kwargs))
  File "/home/chenwy/qdtrack/qdtrack/datasets/coco_video_dataset.py", line 309, in evaluate
    **bbox_kwargs)
  File "/usr/local/lib/python3.6/dist-packages/mmdet/datasets/coco.py", line 413, in evaluate
    result_files, tmp_dir = self.format_results(results, jsonfile_prefix)
  File "/usr/local/lib/python3.6/dist-packages/mmdet/datasets/coco.py", line 358, in format_results
    result_files = self.results2json(results, jsonfile_prefix)
  File "/usr/local/lib/python3.6/dist-packages/mmdet/datasets/coco.py", line 289, in results2json
    if isinstance(results[0], list):
IndexError: list index out of range
```

**Comment by OceanPang** [comment]
@XiaLiPKU Could you please help test the BDD100K config?

**Comment by OceanPang** [comment]
Fixed in #76
